packages/bpk-component-sliding-drawer/src/BpkSlidingDrawer.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Portal } from 'bpk-react-utils';
+import { withScrim } from 'bpk-scrim-utils';
+
+import BpkSlidingDrawerContent from './BpkSlidingDrawerContent';
+
+const ScrimBpkSlidingDrawerContent = withScrim(BpkSlidingDrawerContent);
+
+class BpkSlidingDrawer extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      show: true,
+    };
+
+    this.onExited = this.onExited.bind(this);
+    this.hide = this.hide.bind(this);
+  }
+  onExited() {
```

Comment:
This should be `onExit` to mess well with the React convention. If we want to express that it's called after exit explicitly we should call i `didExit`

Reply:
The callback prop on Transition is called `exited`, figured `onExited` is the best handler name for `exited`